### ISSUE TITLE
Create a claim object in redux and import from query string for now into it

### ISSUE THREAD

**Issue by blurpesec** [ryanjjones10/mm-pay#17]

**Comment by blurpesec** [comment]
https://expo.github.io/router/docs/features/linking/

**Comment by blurpesec** [comment]
For now this will be a b64 encoded/decoded obj:
```typescript
export const testAddr = '0x9c065bdc5a4a9e589Ae7DD555D66E99bb7E9ADe6'

export const testClaim = b64Encode(
  JSON.stringify({
    contractAddress: testAddr,
    privateKey: testPrivateKey,
  }),
)
```

it is dumped in url like so:

http://localhost:19000/?token=eyJjb250cmFjdEFkZHJlc3MiOiIweDljMDY1YmRjNWE0YTllNTg5QWU3REQ1NTVENjZFOTliYjdFOUFEZTYiLCJwcml2YXRlS2V5IjoiMHgwMDAwMTExMTIyMjIzMzMzNDQ0NDU1NTU2NjY2Nzc3Nzg4ODg5OTk5MTAxMDEwMTAxMTExMTExMTEyMTIxMiJ9


This is awaiting a contract & private key mocked by user 1 for user 2

**Comment by blurpesec** [comment]
Closed by #20
